pandas/util/_test_decorators.py

```diff
@@ -312,3 +317,38 @@ def mark_array_manager_not_yet_implemented(request):
     get_option("mode.data_manager") == "array",
     reason="Test that relies on BlockManager internals or specific behaviour",
 )
+
+
+def step_not_implemented(when=None):
```

Comment:
IMO this makes it not entirely obvious that we're testing for the `NotImplemented` step cases. Could you write a unit test function instead e.g. `def test_step_notimplemented<ewm/expanding/cov/corr/groupby/freq>`